backend/spec/features/admin/products/edit/images_spec.rb

```diff
@@ -52,85 +52,6 @@
     end
   end
 
-  context 'Via the upload zone', js: true do
-    before do
-      create(:variant, product: product)
-    end
-
-    it "uploads an image with ajax and appends it to the images table" do
-      visit spree.admin_product_images_path(product)
-      expect(page).to have_content("No images found")
-
-      within_fieldset 'Upload Images' do
-        # Can also pass multiple files in the array, but SQLite gives a deadlock on insert
-        attach_file('image_attachment', [file_path], visible: false)
-      end
-
-      expect(page).not_to have_content("No images found")
-
-      within("table.index") do
-        expect(page).to have_css "tbody tr", count: 1
-
-        within("tbody") do
-          expect(page).to have_xpath "//img[contains(@src,'ror_ringer')]"
-          expect(page).to have_content "All"
-        end
-
-        # Change the image to the other variant
-        targetted_select2 "Size: S", from: "#s2id_image_viewable_id"
-        click_icon :check
-        expect(page).to have_content "Size: S"
-      end
-
-      expect(Spree::Image.last.viewable).to eq(product.master)
-    end
-  end
-
-  it "should see variant images and allow for inline changing the image's variant", js: true do
-    variant = create(:variant)
-    variant.images.create!(attachment: File.open(file_path))
-    visit spree.admin_product_images_path(variant.product)
-
-    expect(page).not_to have_content("No Images Found.")
-
-    within("table.index") do
-      expect(page).to have_content(variant.options_text)
-
-      # ensure no duplicate images are displayed
-      expect(page).to have_css("tbody tr", count: 1)
-
-      # ensure variant header is displayed
-      within("thead") do
-        expect(page).to have_content("Variant")
-      end
-
-      within("tbody") do
-        expect(page).to have_content("Size: S")
-      end
-
-      # Do an inline change of variant and alt
-      targetted_select2 "All", from: "#s2id_image_viewable_id"
-      fill_in 'image[alt]', with: 'ruby on rails t-shirt'
-      click_icon :check
-
-      expect(page).to have_content "All"
-      expect(page).to have_field "image[alt]", with: "ruby on rails t-shirt"
-
-      # test escape
-      find("#image_alt").click # to focus
-      fill_in 'image[alt]', with: 'red shirt'
-      find("#image_alt").send_keys(:escape)
-      expect(page).to have_field "image[alt]", with: "ruby on rails t-shirt"
-
-      # And then go back to Size S variant, but using Enter key
-      targetted_select2 "Size: S", from: "#s2id_image_viewable_id"
-      find("#s2id_image_viewable_id").send_keys(:return)
-
-      expect(page).to have_content "Size: S"
-      expect(page).to have_field "image[alt]", with: "ruby on rails t-shirt"
-    end
-  end
-
```

Comment:
Could we `xit` or even pend these specs and add a note why these fail or don’t test the right thing? I still think this is a great feature, but need a rework someday.

Reply:
This spec didn't work, and as such I think it's best we just remove it for now. There are quite a few areas where we could improve the specs in the backend, and I don't think keeping the code around is going to make this any easier.